zel/src/org/zkoss/zel/ImportHandler.java

```diff
@@ -155,11 +156,25 @@ public void importPackage(String name) {
                         null, "importHandler.invalidPackage", name));
             }
         }
-        packages.add(name);
+        List packageList = importPackages.get(rqpath);
+        if (packageList == null) {
+            packageList = new ArrayList<String>(Arrays.asList(JAVA_LANG_PACKAGE));

```

Comment:
Arrays.asList returns an ArrayList, no need to wrap it again.

Reply:
ok